test/index.test.js

```diff
@@ -46,6 +46,16 @@ describe('Web test', () => {
     expect(response.statusCode).toBe(200)
   })
 
+  test('GET /rules route works', async () => {
```

Comment:
Should we put tests for each route in a dedicated file under `test/routes/`? The healthcheck tests are already there.

Reply:
Yes, good idea.

Reply:
Agreed